src/Our.Umbraco.StackedContent/Web/UI/App_Plugins/StackedContent/js/stackedcontent.controllers.js

```diff
@@ -3,40 +3,40 @@
     "$scope",
     "editorState",
     "notificationsService",
+    "localStorageService",
     "innerContentService",
     "Our.Umbraco.StackedContent.Resources.StackedContentResources",
 
-    function ($scope, editorState, notificationsService, innerContentService, scResources) {
+    function ($scope, editorState, notificationsService, localStorageService, innerContentService, scResources) {
 
         $scope.inited = false;
         $scope.markup = {};
         $scope.prompts = {};
         $scope.model.value = $scope.model.value || [];
 
+        $scope.contentTypeGuids = _.uniq($scope.model.config.contentTypes.map(function (itm) {
+            return itm.icContentTypeGuid;
+        }));
+
         $scope.canAdd = function () {
-            return (!$scope.model.config.maxItems || $scope.model.config.maxItems === 0 || $scope.model.value.length < $scope.model.config.maxItems) && $scope.model.config.singleItemMode !== "1";
+            return (!$scope.model.config.maxItems || $scope.model.config.maxItems === "0" || $scope.model.value.length < $scope.model.config.maxItems) && $scope.model.config.singleItemMode !== "1";
```

Comment:
Is this always going to be string? Should we intParse then comapre?

Reply:
I believe that prevalues will be strings. This wasn't an issue previously, as we were using double-equals, so the JS was evaluating it. I'd tried to follow the ESLint rules by using the triple-equals, but the value was a string (based on my tests).